src/main/scala/com/philcoutinho/main.scala

```diff
@@ -0,0 +1,43 @@
+package com.philcoutinho
+
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import com.philcoutinho.gold.CommentsInTime.commentsintime
+import com.philcoutinho.gold.MediaPreviewLike.mediapreviewlike
+import com.philcoutinho.gold.MediaCommentCount.mediacommentcount
+import com.philcoutinho.gold.MostCommentor.mostcommentor
+import com.philcoutinho.gold.PostTypes.posttypes
+import com.philcoutinho.silver.Posts.posts
+import com.philcoutinho.silver.Profil.profil
+import com.philcoutinho.silver.Comments.comments
+
+
+object main {
+
+  def main(args: Array[String]): Unit = {
+    val spark = SparkSession
+      .builder()
+      .master("local[*]")
+      .appName("main")
+      .getOrCreate()
+    val path = "C:\\Users\\user\\IdeaProjects\\PhilCoutinho_2\\target\\sample.json"
+    val outpath = "C:\\Users\\user\\IdeaProjects\\PhilCoutinho_2\\target\\"
```

Comment:
if you put parquet tables under the target directory, your tables will be pushed if u push the target directory: which shouldn't be the case

Reply:
done